src/java/htsjdk/samtools/CRAMFileWriter.java

```diff
@@ -46,8 +46,9 @@
 
 @SuppressWarnings("UnusedDeclaration")
 public class CRAMFileWriter extends SAMFileWriterImpl {
-    private static final int REF_SEQ_INDEX_NOT_INITIALIZED = -2;
+    private static final int REF_SEQ_INDEX_NOT_INITIALIZED = -3;
```

Comment:
Having looked at the code and spec a bit more closely, and it looks like some of the constants used in Slice are really just alternate names for, and must have the same value as, names already defined in SAMRecord i.e., Slice.UNMAPPED_OR_NO_REFERENCE is really just an alias for SAMRecord.NO_ALIGNMENT_REFERENCE_INDEX and Slice.NO_ALIGNMENT_START is really just an alias for SAMRecord.NO_ALIGNMENT_START. Is that correct ? It it is, can we remove these symbols entirely and just use the SAMRecord constants, or at the very least initialize them in Slice to use the values in SAMRecord to make the relationship explicit?

Maybe others have a different opinion., but for Slice.MULTI_REFERENCE, it would seem strange to me to expose a value through the public interface of SAMRecord that is never used by SAMREcord and is only used by the CRAM code. Since the specific value -2 is prescribed by the CRAM spec, my take would be that it should live in the CRAM package somewhere.